### ISSUE TITLE
Add PG::JSONB converter

### ISSUE THREAD

**Issue by vladfaust** [vladfaust/core#81]
```crystal
# json:
Bytes[123, 34, 102]
# jsonb:
Bytes[1, 123, 34, 102, 111]
```

Thanks @alecdwm for ~~bug~~report!

**Comment by vladfaust** [comment]
Hey @alecdwm could you test the latest `development` branch commit please? I did some refactoring which should not cause any conflicts — in theory :sweat_smile:

**Comment by vladfaust** [comment]
Closed by ecf58ca.

**Comment by alecdwm** [comment]
Seems to work to me, thanks!
